blockchain/receipt.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2014 The go-ethereum Authors
```

Comment:
what is the difference with the original code?

Reply:
only on `BlockNumber` field

Reply:
How about smth like 
```
type SokelReciept struct {
 eth.Receipt
 BlockNumber uint64
}
```

Reply:
Thanks a lot!